### DISCUSSION TITLE
Restore Backup Wallet.dat from Core on Android Wallet

### DISCUSSION THREAD

**Discussion by qlpqlp** [dogecoin/dogecoin#2622] [Help needed] [answered]
Hello again :)

Making a lot of testing, and for now it seams is not possible to restore an wallet.dat Dogecoin Core Backup on Dogecoin Android App, it gives an error.

It should be able on Android to restoure a Backup on File, or Im doing rong? :)

Thanks in advance.

**Comment by patricklodder** [comment, marked as answer]
Those are 2 different applications. You could request this as a new feature at https://github.com/langerhans/dogecoin-wallet-new/issues
